src/DotNetProjectFile.Analyzers/Analyzers/MsBuild/ItemGroupShouldBeUniform.cs

```diff
@@ -0,0 +1,75 @@
+﻿namespace DotNetProjectFile.Analyzers.MsBuild;
+
+[DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
+public sealed class ItemGroupShouldBeUniform : MsBuildProjectFileAnalyzer
+{
+    private static readonly IReadOnlyCollection<IReadOnlyCollection<string>> Exceptions =
+        [
+            [
+                "Compile",
```

Comment:
Might be worth to define these as nodes, and do on the base of types instead.

Reply:
Done